Sources/StreamVideo/Models/CallParticipant.swift

```diff
@@ -23,7 +23,7 @@ public struct CallParticipant: Identifiable, Sendable, Equatable {
     public var hasVideo: Bool
     /// Returns whether the participant has audio.
     public var hasAudio: Bool
-    /// Returns whether the participant is screensharing.
+    /// Returns whether the participant is screenSharing.
```

Comment:
why this change?

Reply:
A few reasons. It's 2 words and capitalising the second word letter makes sense. Xcode was presenting as a typo. Finally it's consistent with the naming of our view (ScreenSharingView).